docs/hopsworksai/azure/restrictive_permissions.md

```diff
@@ -53,10 +64,112 @@ You can then navigate to the JSON tab and overwrite the "action" field with the
     "Microsoft.Resources/subscriptions/resourceGroups/read",
     "Microsoft.Compute/sshPublicKeys/read",
     "Microsoft.ManagedIdentity/userAssignedIdentities/assign/action",
-    "Microsoft.ManagedIdentity/userAssignedIdentities/read"
+    "Microsoft.ManagedIdentity/userAssignedIdentities/read",
+    "Microsoft.Storage/storageAccounts/read",
+    "Microsoft.Compute/snapshots/write",
+    "Microsoft.Compute/snapshots/read",
+    "Microsoft.Compute/snapshots/delete",
+    "Microsoft.Compute/disks/beginGetAccess/action",
+    "Microsoft.Compute/disks/read"
+
   ]
 ```
 
-## Step 4: Create your Hopsworks instance
+### Step 4: Create your Hopsworks instance
 
 You can now create a new Hopsworks instance in Hopsworks.ai by selecting the virtual network, subnet, and network security group during the instance configuration.
+
+### Backup permissions
+
+The following permissions are only needed for the backup feature: 
+
+```json
+"actions": [
+    "Microsoft.Compute/snapshots/write",
+    "Microsoft.Compute/snapshots/read",
+    "Microsoft.Compute/snapshots/delete",
+    "Microsoft.Compute/disks/beginGetAccess/action",
+    "Microsoft.Compute/disks/read"
```

Comment:
@berthoug  "Microsoft.Compute/disks/read" is the permission needed for upgrade, so it would be better to have it as required